Common/utils.ts

```diff
@@ -224,184 +224,9 @@ export function ObjectArrayToCsv(objects: { [key: string]: any }[], headers?: st
-  object1.takedownsInAlcove += NonNone(object2.takedownsInAlcove, 0);
-  object1.takedownsInEnemyFountain += NonNone(object2.takedownsInEnemyFountain, 0);
-  object1.teamBaronKills += NonNone(object2.teamBaronKills, 0);
-  object1.teamDamagePercentage += NonNone(object2.teamDamagePercentage, 0);
-  object1.teamElderDragonKills += NonNone(object2.teamElderDragonKills, 0);
-  object1.teamRiftHeraldKills += NonNone(object2.teamRiftHeraldKills, 0);
-  object1.teleportTakedowns += NonNone(object2.teleportTakedowns, 0);
-  object1.threeWardsOneSweeperCount += NonNone(object2.threeWardsOneSweeperCount, 0);
-  object1.tookLargeDamageSurvived += NonNone(object2.tookLargeDamageSurvived, 0);
-  object1.turretPlatesTaken += NonNone(object2.turretPlatesTaken, 0);
-  object1.turretTakedowns += NonNone(object2.turretTakedowns, 0);
-  object1.turretsTakenWithRiftHerald += NonNone(object2.turretsTakenWithRiftHerald, 0);
-  object1.twentyMinionsIn3SecondsCount += NonNone(object2.twentyMinionsIn3SecondsCount, 0);
-  object1.unseenRecalls += NonNone(object2.unseenRecalls, 0);
-  object1.visionScoreAdvantageLaneOpponent += NonNone(object2.visionScoreAdvantageLaneOpponent, 0);
-  object1.visionScorePerMinute += NonNone(object2.visionScorePerMinute, 0);
-  object1.wardTakedowns += NonNone(object2.wardTakedowns, 0);
-  object1.wardTakedownsBefore20M += NonNone(object2.wardTakedownsBefore20M, 0);
-  object1.wardsGuarded += NonNone(object2.wardsGuarded, 0);
-  object1.xpDiff += NonNone(object2.xpDiff, 0);
-  object1.xpDiff15 += NonNone(object2.xpDiff15, 0);
-  object1.xpDiff25 += NonNone(object2.xpDiff25, 0);
-  object1.goldDiff += NonNone(object2.goldDiff, 0);
-  object1.goldDiff15 += NonNone(object2.goldDiff15, 0);
-  object1.goldDiff25 += NonNone(object2.goldDiff25, 0);
-  object1.csDiff += NonNone(object2.csDiff, 0);
-  object1.csDiff15 += NonNone(object2.csDiff15, 0);
-  object1.csDiff25 += NonNone(object2.csDiff25, 0);
-  object1.totalKillsOfTeam += NonNone(object2.totalKillsOfTeam, 0);
-  object1.totalDeathsOfTeam += NonNone(object2.totalDeathsOfTeam, 0);
-  object1.totalAssistsOfTeam += NonNone(object2.totalAssistsOfTeam, 0);
-  object1.totalDamageDealtToObjectivesOfTeam += NonNone(object2.totalDamageDealtToObjectivesOfTeam, 0);
-  object1.totalGoldOfTeam += NonNone(object2.totalGoldOfTeam, 0);
-  object1.totalPhysicalDamageDealtToChampionsOfTeam += NonNone(object2.totalPhysicalDamageDealtToChampionsOfTeam, 0);
-  object1.totalMagicDamageDealtToChampionsOfTeam += NonNone(object2.totalMagicDamageDealtToChampionsOfTeam, 0);
-  object1.totalTrueDamageDealtToChampionsOfTeam += NonNone(object2.totalTrueDamageDealtToChampionsOfTeam, 0);
-  object1.totalDamageDealtToChampionsOfTeam += NonNone(object2.totalDamageDealtToChampionsOfTeam, 0);
-  object1.totalVisionScoreOfTeam += NonNone(object2.totalVisionScoreOfTeam, 0);
+  for (const key in object1) {
+    // @ts-ignore Typescript being stupid here. Just let me do this
+    object1[key] += NonNone(object2[key], 0);
+  }
```

Comment:
Very Very Very Cool

Reply:
But also what happens if a field in object1 is undefined? It shouldnt be possible but just curious.

Reply:
Then we are fucked anyway. Cuz ur += and undefined